include/spirv-tools/libspirv.h

```diff
@@ -766,6 +768,9 @@ SPIRV_TOOLS_EXPORT void spvDiagnosticDestroy(spv_diagnostic diagnostic);
 SPIRV_TOOLS_EXPORT spv_result_t
 spvDiagnosticPrint(const spv_diagnostic diagnostic);
 
+// Gets the name of an instruction, without the "Op" prefix.
+SPIRV_TOOLS_EXPORT const char* spvOpcodeString(const SpvOp opcode);
```

Comment:
To break the dependency on SpvOp, can we fudge it with uint32_t like this:
 
 // ...
 // Use a uint32_t to avoid a public header dependency on spirv.h
 ....  spvOpcodeStringFromOpcode(uint32_t opcode_value);

Reply:
Good idea.  I've tried it and it works.